pipelines/ApVerifyWithFakes.yaml

```diff
@@ -0,0 +1,49 @@
+# Gen 3 pipeline for ap_verify
+# This concatenates various lsst.verify metrics to an AP pipeline
+
+description: Fully instrumented AP pipeline
+imports:
+    - location: $AP_PIPE_DIR/pipelines/ApPipe.yaml
+    - location: $AP_VERIFY_DIR/pipelines/MetricsRuntime.yaml
+    - location: $AP_VERIFY_DIR/pipelines/MetricsMiscWithFakes.yaml
```

Comment:
It might be more future-proof to import `ApVerify.yaml` here, though I see below that you end up overriding a lot of configs anyway.